### PR TITLE
Prac 05 code review request

### PR DESCRIPTION
**Pull request by TinNilarSoe**
@DianaBukeikhanova
@ChristineYamin
Please check the wimbledon code and emails code.

### CODE REVIEW

File: prac_05/emails.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+def main():
+    """Print user's name and email"""
+    email_to_name = {}
+    email = input("Email: ")
+    while email != "":
+        username = obtain_name_from_email(email)
+        enquiry = input(f"Is your name {username}? (Y/n) ").lower()
+        if enquiry != "y" and enquiry != "":
```
Comment (DianaBukeikhanova) [inline prac_05/emails.py]: It cannot work to full answer (e.g. yes/yea)

Comment (TinNilarSoe) [reply prac_05/emails.py]: Noted. For me, this code is written with the assumption that the user's input can only be Y or enter. Thank you for comment!

Comment (DianaBukeikhanova) [reply prac_05/emails.py]: Good. But be careful.

File: prac_05/wimbledon.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+def process_data(archives):
+    """Create champion_to_count dictionary and countries set"""
+    champion_to_count = {}
+    countries = set()
+    for data in archives:
+        champion_to_count[data[2]] = champion_to_count.get(data[2], 0) + 1
+        countries.add(data[1])
+    return champion_to_count, sorted(countries)
+
+
```
Comment (DianaBukeikhanova) [inline prac_05/wimbledon.py:47]: Good code! Clear.

File: prac_05/word_occurrences.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        word_to_count[word] += 1
+    else:
+        word_to_count[word] = 1
+letters = list(word_to_count.keys())
+max_word_length = max([len(word) for word in letters])
+letters.sort()
+for word in letters:
+    thing, width, other_thing = word, max_word_length, word_to_count[word]
+    print(f"{thing:{width}} : {other_thing}")
+    # print(f"{word:{max_word_length}} : {word_to_count[word]}")
```
Comment (ChristineYamin) [inline prac_05/word_occurrences.py:21]: you can also use eafp way of code, however  your code is nice

File: prac_05/wimbledon.py (hunk 2 above)
Comment (ChristineYamin) [reply prac_05/wimbledon.py:47]: Although this program is hard to write, your code is such a great code.

File: prac_05/hex_colours.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+COLOR_TO_CODE = {"Aliceblue": "#f0f8ff", "Amaranth": "#e52b50", "Beaver": "#9f8170", "Bittersweet": "#fe6f5e",
+                 "Black": "#000000", "Cadetblue": "#5f9ea0", "Cardinal": "#c41e3a", "Daffodil": "#ffff31",
+                 "Dark Byzantium": "#5d3954", "Electric Lime": "#ccff00"}
+color_name = input("Enter the name of the color: ").title()
+while color_name != "":
+    try:
+        print(f"{color_name} is {COLOR_TO_CODE[color_name]}")
+    except KeyError:
+        print("Invalid color")
+    color_name = input("Enter the name of the color: ").title()
```
Comment (ChristineYamin) [inline prac_05/hex_colours.py:10]: clear and easy to understand .

File: prac_05/word_occurrences.py (hunk 3 above)
Comment (TinNilarSoe) [reply prac_05/word_occurrences.py:21]: Thank you.

File: prac_05/wimbledon.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
+"""
+Wimbledon
+Estimate: 40 minutes
+Actual:   40 minutes
+"""
+
```
Comment (ChristineYamin) [inline prac_05/wimbledon.py:6]: It is better when you create a global constant (INDEX-COUNTRY=1 ) 
(INDEX-CHAMPION=2)

Comment (TinNilarSoe) [reply prac_05/wimbledon.py:6]: Thank you. I'll fix it.

**Pull request merged**